Sources/FuzzilliCli/main.swift

```diff
@@ -593,9 +600,17 @@ signalSources.append(source)
 #endif
 
 // Finally, start fuzzing.
-for fuzzer in instances {
-    fuzzer.sync {
-        fuzzer.start(runFor: numIterations)
+let master = fuzzer
```

Comment:
Ah, is this an independent bug fix? Would the thread workers otherwise not get the initial corpus? This should probably be it's own PR as well then

Reply:
The reason this is done, is to synchronize the MAB state and the corpora across each worker.
Without this change, only interesting programs would be distributed to the worker nodes and not the MAB state and compiled seeds.